common/changes/office-ui-fabric-react/u-mahuangh-suggestions-types-rebase_2019-04-10-00-57.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Suggestions: add itemProps type of onRenderSuggestedItem",
+      "type": "patch"
```

Comment:
I think this might be of type `minor` given the _optional_ to _required_ interface change